setup.py

```diff
@@ -22,7 +22,7 @@
     },
     include_package_data=True,
     install_requires=[
-        'django>=1.11,<=3.1',
+        'django>=1.11,<=2.2.19',
```

Comment:
```suggestion
        'django>=1.11,<=3.1.7,
```

@depsiatwal  @webbyfox  How about this instead?